src/CargoSubsystem.cpp

```diff
@@ -27,7 +27,7 @@ void CargoSubsystem::teleop(){
     if(IsIntaking()){
        if(operatorJoystick->GetButton(CORE::COREJoystick::JoystickButton::LEFT_BUTTON)) {
            SetIntake();
-        }else if (operatorJoystick->GetButton(CORE::COREJoystick::JoystickButton::LEFT_TRIGGER)) {
+        }else if (operatorJoystick->GetRisingEdge(CORE::COREJoystick::JoystickButton::LEFT_TRIGGER)) {
```

Comment:
This should not be rising edge, because you want the motors to be running for as long as you are pressing the button, not just when you first pressed it.